src/include/binder/copy/index_look_up_info.h

```diff
@@ -9,12 +9,14 @@ struct IndexLookupInfo {
     common::table_id_t nodeTableID;
     std::shared_ptr<binder::Expression> offset; // output
     std::shared_ptr<binder::Expression> key;    // input
+    std::vector<std::shared_ptr<binder::Expression>> warningData;
```

Comment:
use `binder::expression_vector`. Also I think it's better to call this `warningExprs`. `xxData` we only use in processor.

Reply:
updated